src/opower/opower.py

```diff
@@ -214,6 +215,7 @@ async def async_get_accounts(self) -> list[Account]:
                         customer=Customer(uuid=customer["uuid"]),
                         uuid=account["uuid"],
                         utility_account_id=account["preferredUtilityAccountId"],
+                        service_point_id=account["servicePointId"],

```

Comment:
I do have this field though in utilityAccounts.

Reply:
Ack